metrics-clojure-core/src/metrics/core.clj

```diff
@@ -83,3 +86,28 @@
   "Returns a map of all the counters in the registry and their names."
   [^MetricRegistry reg]
   (.getCounters reg))
+
+(defn uniform-reservior
+  "Create a uniform reservior, which uses Vitter's Algorithm R to
+  produce a statistically representative sample. Default size: 1028."
+  ([] (UniformReservoir.))
```

Comment:
Sorry to nitpick but can we keep the argument vector and function body on separate lines, here and everywhere else?